lib/ansible/modules/files/file.py

```diff
@@ -79,12 +79,13 @@
 '''
 
 EXAMPLES = '''
-# change file ownership, group and mode. When specifying mode using octal numbers, first digit should always be 0.
+# change file ownership, group and mode
 - file:
     path: /etc/foo.conf
     owner: foo
     group: foo
     mode: 0644
+# when specifying mode using octal numbers, add a leading 0
```

Comment:
Just to clarify, I meant for that comment to be /above/ the thing it was commenting. In other words, that comment was meant to refer to the next play. But since this is the second mention of it, maybe it's confusing.

Reply:
I re-modified lib/ansible/modules/files/file.py to move that comment up above the examples.